code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -227,10 +227,21 @@
 	var/base_name = ""
 	var/label_text = ""
 
+/obj/item/storage/pill_bottle/ERT

```

Comment:
This might not be needed either, but if it is, it would be better below `pill_bottle/new` and decapitalize `ERT`.